<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>441</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="datasource_label">
          <property name="text">
           <string>选择数据库:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="datasource_comboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="startDownloadData_button">
          <property name="text">
           <string>更新所有数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="announcement_label">
          <property name="text">
           <string>公告数据:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateAnnouncement_button">
          <property name="text">
           <string>开始更新数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>起始时间:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="announcement_dateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>是否清空数据库:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="isClearAnnDatabase_comboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="announcement_tableView"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="weightData_label">
          <property name="text">
           <string>权重与行业分类数据: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateWeightData_button">
          <property name="text">
           <string>开始更新数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>起始时间:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="weight_dateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>是否清空数据库:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="isClearWeightDatabase_comboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="weightData_tableView"/>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="errorInfo_label">
        <property name="text">
         <string>错误提示信息: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="errorInfo_tableView"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="marketData_label">
          <property name="text">
           <string>历史行情数据:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateHistMarketData_button_">
          <property name="text">
           <string>开始更新数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="marketdata_sttarttime_label">
          <property name="text">
           <string>起始时间:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="marketdata_starttime_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>是否清空数据库:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="isClearHistMarketDatabase_comboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="marketData_tableView"/>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>groupBox</zorder>
   <zorder></zorder>
   <zorder>announcement_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
